### DISCUSSION TITLE
sql restore file path location

### DISCUSSION THREAD

**Discussion by abvural** [jonaski/sqlrestore#3] [Q&A]
Hi Jonas,
I normally have two options when restoring the SQL bak file.
The path to the previous MDF and LDF files or the SQL application's own file path

Can I restore to the original file path where the MDF and LDF files were backed up?

**Comment by jonaski** [comment]
Hi.
No, there is no option for that. So you need to edit the source code.
https://github.com/jonaski/sqlrestore/blob/master/src/backupbackend.cpp#L689
I don't really use this much anymore, since I don't do that kind of work anymore, so I will probably not work on this much.
